solanaj/src/main/java/org/p2p/solanaj/core/PublicKey.kt

```diff
@@ -3,19 +3,20 @@ package org.p2p.solanaj.core
 import org.bitcoinj.core.Base58
 import org.bitcoinj.core.Sha256Hash
 import org.p2p.solanaj.utils.ByteUtils
+import org.p2p.solanaj.utils.PublicKeyValidator
 import org.p2p.solanaj.utils.TweetNaclFast
 import java.io.ByteArrayOutputStream
 
 class PublicKey {
     private var pubkey: ByteArray
```

Comment:
do you mind creating a custom getter here?
`get() = field.copyOf()`
it's better to return copies of byte arrays because they are very very mutable

Reply:
Yes, it's safer to return a copy. This field getter function is `toByteArray()`, I think we could just return the copy here. Also I don't like the `pubkey` naming. What about renaming it to `publicKey`?

Reply:
Oh, I see now.
Hows the idea to rename `toByteArray` to `asByteArray` (meaning that we returning copy of the instance but as ByteArray type. `to` looks for me as we are modifying the instance)

I would absolutely love to see the new `publicKey` name! Thank you.

Reply:
@1sakov what do you think about this naming changes?